tensorflow/compiler/tf2tensorrt/convert/convert_nodes.cc

```diff
@@ -1608,6 +1636,11 @@ tensorflow::Status ConvertConv2DHelper(OpConverterParams* params, int group) {
         node_def.name());
   }
   const nvinfer1::DimsHW dilation(tf_dilations[h_index], tf_dilations[w_index]);
+  if (is_conv2d_backprop_input && (dilation.d[0] != 1 || dilation.d[1] != 1)) {
+    return tensorflow::errors::Unimplemented(
+        "Dilation with Conv2D Transpose is not supported, at ",
```

Comment:
s/Conv2D Transpose/Conv2DBackpropInput? (From end-users point of view conv2d_transpose will be better as that is the API they used but I think they should be able to figure out from Conv2DBackpropInput and the node name.